Fixed NullPointerException on bulk request

catap opened the pull request:
Java's `ArrayList.toArray()` returns provided array when collection is empty.

Here is created a one-element array which contains null element.

Thus, returned `BulkResponse` may contains a null element as `BulkItemResponse`.

How to achieve:
1. Sent a request to `/_bulk?filter_path=took,errors`
2. call inside `BulkProcessor.Listener` a `BulkResponse.hasFailures()`

idegtiarenko commented:
Jenkins test this

idegtiarenko commented on docs/changelog/88358.yaml:
```diff
@@ -0,0 +1,5 @@
+pr: 88385
+summary: Fixed NullPointerException on bulk request
+area: network
```
Hello @catap, thanks for the fix!
Do you mind updating this value to `Distributed` to make the build pass?

```suggestion
area: Distributed
```

catap committed:
Fixed NullPointerException on bulk request

Java's `ArrayList.toArray()` returns provided array when collection is empty.

Here is created a one-element array which contains null element.

Thus, returned `BulkResponse` may contains a null element as `BulkItemResponse`.

How to achieve:
1. Sent a request to `/_bulk?filter_path=took,errors`
2. call inside `BulkProcessor.Listener` a `BulkResponse.hasFailures()`

idegtiarenko commented:
Jenkins test this

idegtiarenko commented:
Thank you!
